packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -727,6 +727,8 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
                     const message1 = this.comparableMessagePayload(this.previouslyProcessedMessage);
                     const message2 = this.comparableMessagePayload(message);
                     if (message1 !== message2) {
+                        // This looks like a data corruption but the culprit has been found instead
```

Comment:
i don't think you can know that for sure. DataCorruptions should be reserved for cases where the persisted data is corrupted. we've seen cases, like db rollback in odsp where this can happen. but re-opening the file is fine.

Reply:
Yeah, makes sense.  I was personally surprised to see it was marked as fileOverwrittenInStorage not dataCorruption - turns out it used to be dataCorruption and Vlad switched it, for I think the case you're talking about.  So I thought I would leave a comment for anyone else wondering the same thing.